package.json

```diff
@@ -30,8 +30,8 @@
     "typescript": "~3.9.7"
   },
   "dependencies": {
-    "@aws-quickstart/eks-blueprints": "^0.1.1",
-    "aws-cdk": "2.17.0",
+    "@aws-quickstart/eks-blueprints": "latest",
```

Comment:
can we use a fix version? last time with use `latest` it breaks backward compatibility...

Reply:
That was the recommended pin by AWS, I am reaching out to see if we can use a fixed version.